### PR TITLE
Create UrlShortener.yml

### PR DESCRIPTION
**Pull request by p53ud0k0d3**

### CODE REVIEW

**Review by p53ud0k0d3 — COMMENTED** [review]
added tags

File: _data/projects/UrlShortener.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- python
+- django
+- urlshortener
+- api
+- html
+- css
+- bootstrap
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/p53ud0k0d3/UrlShortener/issues
```
Comment (shiftkey) [inline _data/projects/UrlShortener.yml]: This should point to a list of issues labelled with whatever you used on the previous line, e.g. https://github.com/p53ud0k0d3/UrlShortener/labels/up-for-grabs

**Review by p53ud0k0d3 — COMMENTED** [review]
corrected link

**Comment by shiftkey** [comment]
@p53ud0k0d3 thanks!

**Pull request merged**